app/(afterLogin)/(feedback)/feedback/settings/define-feedback/page.tsx

```
@@ -80,8 +80,14 @@ const Page = () => {
     return departments?.find((item: Department) => item.id === id);
   };
   useEffect(() => {
-    setActiveTab(getAllFeedbackTypes?.items?.[0]?.id);
-  }, [getAllFeedbackTypes]);
+    // Only set activeTab if it's not already set or if the current activeTab is not valid
+    if (getAllFeedbackTypes?.items?.length > 0) {
+      const isValidActiveTab = getAllFeedbackTypes.items.some((item: FeedbackTypeItems) => item.id === activeTab);
```

ESLint

[error] Replace `(item:·FeedbackTypeItems)·=>·item.id·===·activeTab` with `⏎········(item:·FeedbackTypeItems)·=>·item.id·===·activeTab,⏎······`

(prettier/prettier)